app/routes/card.py

```diff
@@ -11,3 +11,34 @@
 # GET /boards/<board_id>/cards
 
 # POST /boards/<board_id>/cards
+
+#DELETE
+@bp.route("/<card_id>", methods = ["DELETE"])
+def delete_card(card_id):
+    
+    if not card_id.isnumeric():
+        return { "Error": "Card id must be numeric."}, 404
+    card_id = int(card_id)
+    card = Card.query.get(card_id)
+    
+    if not card:
+        return "Card does not exist",404
+            
+    db.session.delete(card)
+    db.session.commit()
+
+    return {
+        'message': f'Card {card.card_id} from the board {card.board_id} was successfully deleted'
+    }, 200
+
+#LIKE PUT
+@bp.route("/<card_id>/like", methods = ["PUT"])
+def like_card(card_id):
+    card_id = int(card_id)
+    card = Card.query.get(card_id)
+    card.likes_count +=1
```

Comment:
Yes! This looks amazing. I hadn't put a likes_count on my fake cards and this +=1 threw a 500 error. I think I would prefer to mandate a default value of 0 for likes_counts in the Card model rather than coding logic in the route. What do y'all think?

Reply:
Nevermind, I see that you already accounted for this above, with the likes_count = 0 routes/boards.py line 65 :D